docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,511 @@
+{
```

Comment:
<blockquote>... patterns(PVCs) ...</blockquote><p> </p><p>... patterns (PVCs) ... [Notice the space I added]</p><p> </p><p> </p><p>Also, what is PVC? should reader know about this? If not, then you can/should remove it. If yes, then you should avoid using abbreviation unless you are going to use it again later. Even in such case, you should first provide the original phrase. </p><p> </p><blockquote>Those are target we try to find. </blockquote><p> </p><p>Those are the targets we try to find.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I removed the abbreviation.